suites/reef/rados/tier-2_rados_test-pg-dups-trimming.yaml

```diff
@@ -1,4 +1,5 @@
 # Suite is to be used to verify the dups trimming scenario on all NVMe cluster
+# Conf file to be used : conf/quincy/baremetal_pipeline/mero_conf.yaml
```

Comment:
Please change to `conf/reef/baremetal/mero_conf.yaml`

Reply:
Updated!